client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -209,6 +209,36 @@ const PluginMeta = React.createClass( {
 		}
 	},
 
+	maybeDisplayUnsupportedNotice() {
+		const { selectedSite, plugin } = this.props;
+
+		// Pressable prevents installation of some plugins, so we need to disable AT for them.
+		// More info here: https://kb.pressable.com/faq/does-pressable-restrict-any-plugins/
+		const unsupportedPlugins = [
+			'nginx-helper',
+			'w3-total-cache',
+			'wp-rocket',
+			'wp-super-cache',
+			'bwp-minify',
+		];
+
+		if ( selectedSite && ! selectedSite.jetpack && includes( unsupportedPlugins, plugin.slug ) ) {
```

Comment:
Updated to account for already transferred sites in 6719d7f90c36092f09dbe86f824883866c2f1a53.